tests/test_fixtures.bash

```diff
@@ -19,10 +19,11 @@ function setup_file() {
     export INPUT_DOMAIN=gh_actions_host
     export INPUT_FAIL_FAST=false
     export INPUT_INDEX=false
+    export INPUT_CREATE_OUTPUT=false
     export INPUT_PATH=/
     export INPUT_PROTOCOL=http
     export INPUT_SKIP_OPTIONAL=false
-    export INPUT_VALIDATOR_VERSION=latest
+    export INPUT_VALIDATOR_VERSION=master
```

Comment:
Can this go back to testing latest now 0.12.1 is out? (suggesting change just for your convenience)

```suggestion
    export INPUT_VALIDATOR_VERSION=latest
```